node_modules/oae-util/tests/test-globals.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var assert = require('assert');
+
+describe('Globals', function() {
+
+    describe('Underscore', function() {
+
+        describe('oaeObj', function() {
+
+            /**
+             * Test that verifies that an object is created from a variety of input
+             */
+            it('verify an object is created from a variety of input', function() {
```

Comment:
I personally am fine with not adding `callback` in the arguments and not doing `return callback();` for synchronous tests, but I believe we settled on doing so